src/Sylius/Bundle/AdminBundle/Resources/views/Promotion/_form.html.twig

```diff
@@ -23,8 +23,8 @@
         <h4 class="ui dividing header">{{ 'sylius.ui.start_date'|trans }} & {{ 'sylius.ui.end_date'|trans }}</h4>
 
         <div class="two fields">
-            {{ form_row(form.startsAt) }}
-            {{ form_row(form.endsAt) }}
+            {{ form_row(form.startsAt, { 'id': 'rangestart'}) }}
+            {{ form_row(form.endsAt, { 'id': 'rangeend'}) }}
```

Comment:
The default calendar is rendered without anything. The case of this `id` is to define with which other calendar it should be coupled to present a range of dates. I didn't have better idea for this.